openmetadata-ui/src/main/resources/ui/cypress/common/Utils/AdvancedSearch.ts

```diff
@@ -75,13 +75,153 @@ export const ADVANCE_SEARCH_TABLES = {
   },
 };
 
+export const DATABASE_SERVICE_DETAILS_2 = {
+  name: `cy-service-as2-${uuid()}`,
+  serviceType: 'Mysql',
+  connection: {
+    config: {
+      type: 'Mysql',
+      scheme: 'mysql+pymysql',
+      username: 'username',
+      authType: {
+        password: 'password',
+      },
+      hostPort: 'mysql:3306',
+      supportsMetadataExtraction: true,
+      supportsDBTExtraction: true,
+      supportsProfiler: true,
+      supportsQueryComment: true,
+    },
+  },
+};
```

Comment:
instead of this can we create one function that will create a random service, or just pass the name of the service and it will give a full service object?
or we can create full hierarchy, and function will return object containing, 
```
{
service: <DATA>,
schema: <DATA>,
database: <DATA>,
table: <DATA>,
}
```